### PR TITLE
Teach the manifest about dictionaries

### LINKED ISSUE #393: Rewriting fastly.toml strips out some properties
**Version**

Fastly CLI version 0.37.1 (e100937)

**What happened**

When running `fastly compute serve` I noticed dictionary lookups were causing panics. 
Upon further inspection, I found that the CLI was stripping out some properties (e.g. local_server.dictionaries) when rewriting the fastly.toml and the original input did not have `manifest_version`.

fwiw, the same file works without changes in Viceroy.

### PR DESCRIPTION
**Pull request by fgsch**
Fixes #393.

### CODE REVIEW

File: pkg/commands/compute/manifest/manifest.go [hunk 1]
```diff
@@ -225,6 +226,12 @@ type Backend struct {
 	OverrideHost string `toml:"override_host,omitempty"`
 }
 
+// Dictionary represents a dictionary to be mocked by the local testing server.
+type Dictionary struct {
+	File   string `toml:"file"`
+	Format string `toml:"format"`
```
Comment (Integralist) [inline pkg/commands/compute/manifest/manifest.go:232]: Does `format` need `omitempty`? 

By this I mean: is it a required property, or will the file format be inferred (or a default used) by Viceroy?

**Review by Integralist — APPROVED** [review]
LGTM. One small comment.

File: pkg/commands/compute/manifest/manifest.go (hunk 1 above)
Comment (fgsch) [reply pkg/commands/compute/manifest/manifest.go:232]: It is required.

Comment (Integralist) [reply pkg/commands/compute/manifest/manifest.go:232]: OK cool 👍🏻

**Pull request merged**